docs/setup_config.rst

```diff
@@ -0,0 +1,58 @@
+Setup configuration
+===================
+
+Loading notification configuration from a YAML file
+***************************************************
+
+This library provides two ``ConfigurationStep``s
```

Comment:
Formatting seems to crash with characters directly after ``
![image](https://github.com/user-attachments/assets/91d08706-7dfb-4012-86bc-e6f63367f055)

```suggestion
This library provides two ``ConfigurationSteps``
```